pkg/webhooks/group/group.go

```diff
@@ -36,15 +36,17 @@ type groupRequest struct {
 }
 
 const (
-	WebhookName     string = "group-validation"
-	protectedGroups string = `(^osd.*|^dedicated-admins$|^cluster-admins$|^layered-cs-sre-admins$)`
+	WebhookName               string = "group-validation"
+	protectedAdminGroups      string = `(^dedicated-admins$|^cluster-admins$)`
+	protectedManagementGroups string = `(^osd.*|^layered-cs-sre-admins$)`
```

Comment:
```suggestion
	protectedManagementGroups string = `(^osd-sre-admins$|^osd-sre-cluster-admins$|^osd-devaccess$|^layered-cs-sre-admins$)`
```

I think we should hard-code the 3 specific groups here, so that customers can create other groups that start with osd if they want to

Reply:
:+1: to this. I saw in the ticket there should be conversation about it, so I figured PR is where we should